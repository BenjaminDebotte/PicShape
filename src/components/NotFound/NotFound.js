import React from 'react';

const NotFound = (props) => {
  return (
    <div className="container text-center">
      <img src={require('../../images/Modified404.png')}/>
   </div>
  );
};

export default NotFound;
