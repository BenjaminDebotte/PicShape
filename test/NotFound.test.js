import React from 'react';
import  {NotFound} from '../src/components/NotFound/NotFound';
import renderer from 'react-test-renderer';

describe('components', function() {
  describe('<NotFound />', function() {
    it('renders correctly', function() {
      var tree = renderer.create(<NotFound />).toJSON();
      expect(tree).toMatchSnapshot();
        });
  });
});
