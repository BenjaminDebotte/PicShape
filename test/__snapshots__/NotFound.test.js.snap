exports[`components <NotFound /> renders correctly 1`] = `
<div
  className="ui container text-center">
  <h1>
    404
  </h1>
  <p>
    Page Not Found
  </p>
</div>
`;
